server/migration/migration.js

```diff
@@ -13,17 +13,30 @@ const createTable = pool.query(
     is_admin BOOLEAN NOT NULL DEFAULT false,
     address VARCHAR(255),
     created_at  DATE NOT NULL,
-    updated_at  DATE NOT NULL);;
+    updated_at  DATE NOT NULL);
 
   CREATE TABLE IF NOT EXISTS gifs (
         gif_id SERIAL PRIMARY KEY,
         title VARCHAR(128) NOT NULL,
         image_url VARCHAR(128) NOT NULL,
-        public_id varchar NOT NULL,
+        public_id VARCHAR NOT NULL,
         created_at timestamp with time zone NOT NULL,
         user_id SERIAL NOT NULL,
         FOREIGN KEY (user_id) 
         REFERENCES users (id));
-        
+      
+    CREATE TABLE IF NOT EXISTS gif_comment (
+          gif_id INTEGER NOT NULL,
+          comment_id SERIAL PRIMARY KEY,
+          comments VARCHAR NOT NULL,        
+          created_at timestamp with time zone NOT NULL,
+          created_by VARCHAR NOT NULL,
```

Comment:
we dont need created by and why is the gig id also a foreign key referencing the gif table

Reply:
Ok sir. I have removed the created_by across all routes